cli/tests/test_commit_template.rs

```diff
@@ -230,6 +230,72 @@ fn test_log_builtin_templates() {
     "###);
 }
 
+#[test]
+fn test_log_builtin_templates_colored() {
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_success(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+    let render =
+        |template| test_env.jj_cmd_success(&repo_path, &["--color=always", "log", "-T", template]);
+
+    test_env.jj_cmd_ok(
+        &repo_path,
+        &[
+            "--config-toml=user.email=''",
+            "--config-toml=user.name=''",
+            "new",
+        ],
+    );
+
+    insta::assert_snapshot!(render(r#"builtin_log_oneline"#), @r###"
```

Comment:
Nit: I don't think this is important for this PR, but I find it nice to first have a commit adding the test, and then have a separate commit making the change (to the template, in this case) and thus changing the test.

Reply:
Makes sense, done.